framework/validators/Validator.php

```diff
@@ -415,4 +419,26 @@ public function isEmpty($value)
             return $value === null || $value === [] || $value === '';
         }
     }
+
+    /**
+     * @return array
```

Comment:
Need `@since <version>` tag where `<version>` is Yii version it's merged in.

Also need proper phpdoc.